pudb/debugger.py

```diff
@@ -780,7 +780,7 @@ def change_var_state(w, size, key):
             iinfo = self.get_frame_var_info(read_only=False) \
                     .get_inspect_info(var.id_path, read_only=False)
 
-            if key == "\\" or key == ' ':
+            if key == "enter" or key == "\\" or key == ' ':
```

Comment:
Could you update the help text to reflect these changes?

Reply:
Sorry about that. Done now.